site/mex_invenio/config/settings.py

```diff
@@ -254,6 +254,38 @@ def _(x):  # needed to avoid start time failure with lazy strings
 # if it is not present in the MEx source record
 RECORD_METADATA_DEFAULT_TITLE = "[Untitled]"
 
+# ---
+# Custom facets
+# ---
+
+from invenio_rdm_records.config import RDM_SEARCH
+from mex_invenio.custom_facets import RestrictedTermsFacet
+from invenio_vocabularies.services.facets import VocabularyLabels
+
+# Define a new facet that only shows "Documentation" and "Variable"
+RDM_FACETS = {
+    "restricted_resource_type": {
+        "facet": RestrictedTermsFacet(
+            field="metadata.resource_type.props.type",
+            subfield="metadata.resource_type.props.subtype",
+            splitchar="::",
+            # Field to filter on
+            label=_("Resource types"),
+            value_labels=VocabularyLabels("resourcetypes"),
+            allowed_values=["resource", "bibliographicresource", "variable", "activity"]
+        ),
+        "ui": {
+            "field": "resource_type.id"
+        },
+    },
+}
+
+# Add the new facet to the search configuration
+RDM_SEARCH = {
+    **RDM_SEARCH,
```

Comment:
I'm not sure what including `**RDM_SEARCH` does for us here  since we're overwriting to remove the default facets in favour of `restricted_resource_type`

Reply:
It contains also other settings, for example sorting.